src/parser/jobs/mnk/modules/Forms.tsx

```diff
@@ -85,12 +86,12 @@ export default class Forms extends Module {
 			// Using Opo-Opo skills resets form
 			case STATUSES.RAPTOR_FORM.id:
 			case STATUSES.COEURL_FORM.id:
-				if (OPO_OPO_SKILLS.includes(action)) { this.resetForms++ }
+				if (OPO_OPO_SKILLS.includes(action.id)) { this.resetForms++ }
 				break
 
 			default:
 				// No form used
-				if (OPO_OPO_SKILLS.includes(action)) {
+				if (OPO_OPO_SKILLS.includes(action.id)) {
 					this.formless++
```

Comment:
Does this need a check for whether they're under perfect balance?  Showing 19 formless skill usages for http://localhost:3000/analyse/BM9hJdHwpxfFN6qg/11/6/ (compared to no suggestion being made on live), and looking at the ones that are detecting as formless, 16 of them occurred while under the Perfect Balance status.

Also, the recommendation specifically says "Avoid Using Dragon Kick outside of Opo-Opo Form", but this is going to ding if any of the three Opo-Opo skills are used formless, is that intended?  If so, should the recommendation copy be updated?

Reply:
I think this was just broken before. It does need a PB check, but I'm wondering if that will make hide bad dropped forms due to bad PB timing since the check would need to be before the form switch. I'm not sure that really matters tho besides in the opener, but we do kinda care what GCD was used immediately before a PB I guess. Maybe a job for the PB module tho, and thinking of doing similar in RoF and adding BH at some point. Prob need to talk to some expert monks.